batch/batch/batch.py

```diff
@@ -314,10 +314,11 @@ def _keep_alive_container(self):  # pylint: disable=R0201
             self.log_info(f'ignoring: could not delete {self._pvc_name} due to {err}')
 
     async def _delete_pod(self):
-        err = await app['k8s'].delete_pod(name=self._pod_name)
-        if err is not None:
-            traceback.print_tb(err.__traceback__)
-            self.log_info(f'ignoring pod deletion failure due to {err}')
+        pass
+        # err = await app['k8s'].delete_pod(name=self._pod_name)
+        # if err is not None:
+        #     traceback.print_tb(err.__traceback__)
+        #     self.log_info(f'ignoring pod deletion failure due to {err}')
```

Comment:
Did you mean to leave this in?

Reply:
It's the only way I can get the logs for failing pods right now. I'll delete later. I have to keep this as a PR because it works in my namespace since batch-pods == default in dev deploy.

Reply:
Oh, yeah, sure, no worries.